delete_db.js

```diff
@@ -0,0 +1,11 @@
+import pg from 'pg';
```

Comment:
Wouldn't it be cleaner to use mocha's before() and after() functions to setup/teardown the DB, rather than having `create_db.js` and `delete_db.js`?

Reply:
No. Mocha runs its tests on a block by block (File) basis and gets ugly when you try to do global before and afters. It's allot cleaner to have it as a separate task (I had it mostly working with mocha before and switched it to this) 

This also leaves the ability to update and expand these scripts to be more useful.

Reply:
Sounds good.